README.md

```diff
@@ -244,16 +244,23 @@ Companies that have allowed us to speak about our work can be found here. Many m
 
 | Product | Date | Level of <br />Effort | Announcement | Report |
 | --- | --: | :-: | --- | :-: |
+| [OpenVPN3](https://openvpn.net/) | Jan 2023 | 6 |  |  |
+| [Fraxlend and veFPIS](https://frax.finance/) | Jan 2023 | 4 |  |  |
+| [Redpanda Core, Console, and Console Enterprise](https://redpanda.com/) | Jan 2023 | 4 |  |  |
+| [Injective Labs Options Market](https://injective.com/) | Jan 2023 | 4 |  |  |
+| [OpenArchive (Android)](https://open-archive.org/) | Dec 2022 | 1 |  |  |
+| [Enclave Markets Trading Platform](https://www.enclave.market/) | Nov 2022 | 9 |  |  |
 | [Phantom](https://phantom.app/) | Nov 2022 | 2 |  |  |
 | [Fiat Ramps](https://cashero.com/) | Nov 2022 | 4 |  |  |
-| [cURL](https://curl.se/) | Oct 2022 | 9.5 | [OSTIF (official announcement)](https://ostif.org/the-ostif-audit-of-curl-with-trail-of-bits-is-complete/), [Daniel Stenberg (blog)](https://daniel.haxx.se/blog/2022/12/21/the-2022-curl-security-audit/). [Trail of Bits (blog)](https://blog.trailofbits.com/2022/12/22/curl-security-audit-threat-model/) | [📄](reviews/2022-12-curl-securityreview.pdf) [📛](reviews/2022-12-curl-threatmodel.pdf) |
+| [cURL](https://curl.se/) | Oct 2022 | 9.5 | [OSTIF (official announcement)](https://ostif.org/the-ostif-audit-of-curl-with-trail-of-bits-is-complete/), [Daniel Stenberg (blog)](https://daniel.haxx.se/blog/2022/12/21/the-2022-curl-security-audit/). [Trail of Bits (blog)](https://blog.trailofbits.com/2022/12/22/curl-security-audit-threat-model/) | [📄✅](reviews/2022-12-curl-securityreview.pdf)[📛](reviews/2022-12-curl-threatmodel.pdf) |
 | [CloudEvents](https://cloudevents.io/) | Oct 2022 | 4 | [Results of CloudEvents Security<br>Assessment](https://ostif.org/results-of-the-cloudevents-security-assessment/) | [📄](reviews/CloudEvents.pdf) | 
 | [OpenArchive Save (iOS)](https://open-archive.org/) | Oct 2022 | 1.2 |  |  |
-| [SimpleX Chat](https://simplex.chat/) | Oct 2022 | 1 | | [📄](reviews/SimpleXChat.pdf) | 
-| [AlphaSOC API](https://alphasoc.com/) | Sep 2022 | 1 |  |  |
+| [Fraxlend and FraxFerry](https://frax.finance/) | Oct 2022 | 4 |  |  |
```

Comment:
Missing the final report link https://drive.google.com/file/d/1h82oLgUXds0Z-6k-rSyGl8qzcI9adpGJ/view?usp=share_link

Reply:
Great catch! I had added the report to the `reviews/` folder but missed adding this link somehow. Thanks!